datacube_stats/mangrove.py

```diff
@@ -0,0 +1,69 @@
+import logging
+import xarray as xr
+import numpy as np
+from osgeo import ogr
+from osgeo import gdal
+from osgeo import osr
+from datacube.model import Measurement
+from .statistics import Statistic
+
+_LOG = logging.getLogger(__name__)
+
+
+class MangroveCC(Statistic):
+    def __init__(self, thresholds, shape_file, bands=None):
```

Comment:
Q: is this `shape_file` the same as the `input_region`?

Reply:
No, the input_region is taking the tile-file and this shape_file is the parameter for the raster purely.